helpers/calendar/push_notifications.py

```diff
@@ -0,0 +1,49 @@
+import graphene
+from helpers.calendar.credentials import Credentials
+from flask import jsonify
+from pyfcm import FCMNotification
+import uuid
+import os
+
+notification_url = os.getenv('NOTIFICATION_URL') if os.getenv('APP_SETTINGS') == 'production' else os.getenv('DEV_NOTIFICATION_URL')  # noqa
```

Comment:
Is there a better way to break up this conditional statement into multiple lines to prevent the use of `noqa`

Reply:
For this I think creating  assigning the environment variables to other shorter named variables should help you avoid noqa

Reply:
I tried to avoid abbreviating those variable names. I would wrap that line in braces and break into multiple lines.